theSrc/scripts/PlotData.coffee

```diff
@@ -445,7 +444,7 @@ class PlotData
     }
     @outsidePlotPtsId.push id
     @normalizeData()
-    @getPtsAndLabs()
+    @getPtsAndLabs('addElemFromLegend')
```

Comment:
no that is left over debug code - I was tracking down why it was being called too many times -
 will delete